workflow-core/build.gradle.kts

```diff
@@ -5,28 +5,15 @@ plugins {
 
 kotlin {
   jvm { withJava() }
```

Comment:
Could you add a quick comment here to look at the `kotlin-multiplatform` include for the other KMP targets?

Reply:
Imported the function from buildSrc instead.